templates/gRPC-C++.podspec.template

```diff
@@ -60,42 +60,43 @@
         files.update(lib.get(field, []))
     return list(sorted(files))
 
-  def list_filegroup_files(expect_filegroups, groups):
-    out = []
-    for filegroup_name in expect_filegroups:
-      filegroup = filegroup_maps[filegroup_name]
-      for group in groups:
-        out += filegroup.get(group, [])
-    return list(sorted(set(out)))
-
   def filter_grpcpp(files):
     return sorted([file for file in files if not file.startswith("include/grpc++")])
 
-  grpc_private_files = list_lib_files("grpc", ("headers", "src"))
-  grpc_public_headers = list_lib_files("grpc", ("public_headers",))
-  grpc_private_headers = list_lib_files("grpc", ("headers",))
+  # TODO(jtattermusch): get rid of this hack
+  address_sorting_unwanted_files = list_lib_files("address_sorting", ("public_headers", "headers", "src"))
+
+  grpc_private_files = list(sorted(set(list_lib_files("grpc", ("headers", "src"))) - set(address_sorting_unwanted_files)))
+  grpc_public_headers = list(sorted(set(list_lib_files("grpc", ("public_headers",))) - set(address_sorting_unwanted_files)))
+  grpc_private_headers = list(sorted(set(list_lib_files("grpc", ("headers",))) - set(address_sorting_unwanted_files)))
+
+  # TODO(jtattermusch): build.yaml no longer has filegroups, so the files here are just hand-listed
```

Comment:
I feel this makes it more than a bit fragile. The podspec templates were written so that build.yaml is the source of truth for the file list, and this change seems to be breaking that. If someone adds another proto related header later, I wonder if they will know what this is about and make corresponding corrections here. 

Is it possible to add things in build_handwritten (because it's hand written)?

Reply:
Potentially we could add this in build_handwritten.yaml, I'll look into that.

I understand the desire to have the list of file for grpc_cronet in a special place (I'll try to clean up), but in practice this approach this doesn't really change much IMHO. AFAIK The grpc_cronet was only built as part of the objC build even before and the podspecs were hand-massaging and filtering the list of the files anyway.